### ISSUE TITLE
Error while trying to deploy

### ISSUE THREAD

**Issue by jbblanchet** [DataDog/heroku-buildpack-datadog#34]
I'm having the following error when trying to deploy to an heroku app with the datadog buildpack since yesterday evening:

```
remote: -----> Build succeeded!        
remote: -----> Sysstat app detected        
remote: -----> Updating apt caches for dependencies        
remote:        Get:1 http://apt.postgresql.org/pub/repos/apt xenial-pgdg InRelease [46.4 kB]        
remote:        Get:2 http://archive.ubuntu.com/ubuntu xenial InRelease [247 kB]        
remote:        Get:3 http://apt.postgresql.org/pub/repos/apt xenial-pgdg/main amd64 Packages [145 kB]        
remote:        Get:4 http://archive.ubuntu.com/ubuntu xenial-security InRelease [102 kB]        
remote:        Get:5 http://archive.ubuntu.com/ubuntu xenial-updates InRelease [102 kB]        
remote:        Get:6 http://archive.ubuntu.com/ubuntu xenial/main amd64 Packages [1,558 kB]        
remote:        Get:7 http://archive.ubuntu.com/ubuntu xenial/universe amd64 Packages [9,827 kB]        
remote:        Get:8 http://archive.ubuntu.com/ubuntu xenial-security/main amd64 Packages [585 kB]        
remote:        Get:9 http://archive.ubuntu.com/ubuntu xenial-security/universe amd64 Packages [405 kB]        
remote:        Get:10 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 Packages [953 kB]        
remote:        Get:11 http://archive.ubuntu.com/ubuntu xenial-updates/universe amd64 Packages [762 kB]        
remote:        Fetched 14.7 MB in 2s (5,086 kB/s)        
remote:        Reading package lists...        
remote: -----> Installing dependencies        
remote:        Reading package lists...        
remote:        Building dependency tree...        
remote:        Suggested packages:        
remote:          lm-sensors snmp-mibs-downloader        
remote:        The following NEW packages will be installed:        
remote:          libpci-dev libpci3 libsensors4 libsensors4-dev libsnmp-base libsnmp30        
remote:        0 upgraded, 6 newly installed, 0 to remove and 70 not upgraded.        
remote:        Need to get 1,035 kB/1,164 kB of archives.        
remote:        After this operation, 4,725 kB of additional disk space will be used.        
remote:        Get:1 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libsnmp-base all 5.7.3+dfsg-1ubuntu4.1 [224 kB]        
remote:        Get:2 http://archive.ubuntu.com/ubuntu xenial-updates/main amd64 libsnmp30 amd64 5.7.3+dfsg-1ubuntu4.1 [811 kB]        
remote:        Fetched 1,035 kB in 0s (1,402 kB/s)        
remote:        Download complete and in download only mode        
remote: W: --force-yes is deprecated, use one of the options starting with --allow instead.        
remote:        Installing libpci-dev        
remote:        Installing libpci3        
remote:        Installing libsensors4        
remote:        Installing libsensors4-dev        
remote:        Installing libsnmp-base        
remote: dpkg-deb: error: --extract takes at most two arguments (.deb and directory)        
remote: 
remote: Type dpkg-deb --help for help about manipulating *.deb files;        
remote: Type dpkg --help for help about installing and deinstalling packages.        
remote:  !     Push rejected, failed to compile Sysstat app.        
remote: 
remote:  !     Push failed        
```

**Comment by xACruceSalus** [comment]
Just got the same thing. When I remove this buildpack, it works fine. As mentioned above, it was working a few days ago though.

**Comment by jyee** [comment]
This should be fixed by #33

**Comment by jbblanchet** [comment]
Just tested it and it worked. Thanks @jyee for the quick fix.
